### PR TITLE
add pluralize to funcs (fix #51)

### LINKED ISSUE #51: Please export Pluralize to funcs
.FuncName already Pluralize to particular use case, but that it does not cover some use case like search by multiple composite indexes. Thanks.

### PR DESCRIPTION
**Pull request by S-YOU**

### CODE REVIEW

**Comment by S-YOU** [comment]
I am not very confident about this is the correct way to implement. Please take a look when you have time @kazegusuri 
Thanks.

File: generator/funcs.go [hunk 1]
```diff
@@ -639,3 +641,8 @@ func (a *Generator) nullcheck(field *internal.Field) string {
 
 	return fmt.Sprintf("yo, ok := %s.(yoIsNull); ok && yo.IsNull()", paramName)
 }
+
+// pluralize converts s to pural.
```
Comment (kazegusuri) [inline generator/funcs.go:645]: typo: plural

Comment (S-YOU) [reply generator/funcs.go]: Thanks. Fixed @kazegusuri

**Comment by kazegusuri** [comment]
LGTM. could you squash the commits?

**Comment by S-YOU** [comment]
> LGTM. could you squash the commits?

Sure, done. Thanks @kazegusuri

**Review by kazegusuri — APPROVED** [review]

**Comment by S-YOU** [comment]
It's been a while. Could you please release new version with this? @kazegusuri

**Pull request merged**